src/components/ChooseToken.vue

```diff
@@ -220,8 +225,10 @@ export default Vue.extend({
       }
       return this.$emit("chosen", symbolOrID);
     },
-    parseBigNumber(value: string) {
-      return formatFixed(value, 18);
```

Comment:
Isn't this approach better as it doesn't rely on importing new dependencies into the project?

Reply:
Being that, it's safer to use a proper Big number library for this kind of operations, I believe we can move forward